apps/system/test/unit/lockscreen/widgets/clock/lockscreen_clock_widget_test.js

```diff
@@ -41,12 +40,12 @@ suite('LockScreenClockWidget > ', function() {
       }}
     };
     var method = LockScreenClockWidget.prototype.updateClock;
+    var now = new Date();
     method.call(mockThis);
-    assert.equal(navigator.mozL10n.dummyLocaleFormat(),
+    assert.equal(mockThis.timeFormatter.format(now),
```

Comment:
Why should it not match?

the call to `method.call(mockThis)` changes the `mockThis.resources.elements.date.textContent` to the same value as returned by `mockThis.timeFormatter.format(now)`

I know Intl is fairly new (not *that* new, we're just starting using it ;)), but since it's a standard built-in object I like to think about testing it the same way as I would test an Array or Object.

I'd call the function that I'm testing and then compare if the property it modified has the expected value. Which I'm doing here. Does it make sense?